src/Database/Driver/Sqlite.php

```diff
@@ -176,11 +183,30 @@ public function prepare($query): StatementInterface
         return $result;
     }
 
+    /**
+     * Query SQLite DB for current status of foreign keys support
+     *
+     * @return  bool      True if foreign key support is currently enabled
+     */
+    public function isForeignKeyEnabled(): bool
```

Comment:
We don't generally add new public methods in bug fix releases. This would need to be a protected method or moved to 4.3.

Reply:
I understand. It can easily be turned into a protected method as it was mainly made public for test purpose. The return value of `SqliteDriver::enableForeignKeySQL` is enough to check for validity.